lib/ansible/galaxy/role.py

```diff
@@ -207,10 +207,73 @@ def fetch(self, role_data):
                 temp_file.close()
                 return temp_file.name
             except Exception as e:
-                display.error(u"failed to download the file: %s" % to_text(e))
```

Comment:
This change looks wrong: `to_text()` always returns a Unicode string so rendering it into a native string (which is `bytes` on Python 2) seems unwanted.

Reply:
That was already present, will change that

Reply:
Changed line by accident, will revert